continuous_load_testing/client.go

```diff
@@ -263,8 +284,47 @@ func ExecuteHalfDuplexCalls(ctx context.Context, tc test.TestServiceClient) erro
 	return nil
 }
 
+func ExecuteStreamedSequentialUnaryCall(ctx context.Context, tc test.TestServiceClient) error {
+	var stream test.TestService_StreamedSequentialUnaryCallClient
+	var err error
+	// Create the bidi streaming connection
+	for {
+		stream, err = tc.StreamedSequentialUnaryCall(ctx)
+		if err != nil {
```

Comment:
for stream, err = tc.StreamSequentialUnaryCall(ctx); err != nil; {
    // handle err and backoff
}